### ISSUE TITLE
Loading xml content into graph

### ISSUE THREAD

**Issue by tacman** [njh/easyrdf#313]
I'm trying to figure out how to load XML into a graph

```php
        $url = "http://njh.me/foaf.rdf";
        $foaf = new Graph($url);
        $foaf->load();
        $me = $foaf->primaryTopic();
        // $me is set!

        $contents = file_get_contents($url);
        $graph = new Graph();
        $response = $graph->parse($contents, $format, $url);
        $me = $graph->primaryTopic();
        if (!$me) { die('$me is not set when loading via parser.');} // but this works
```

I imagine it's something simple, like not calling the loader when parsing content from a string.  Thanks.

**Comment by njh** [comment]
Is it an RDF/XML file stored on disc?

**Comment by tacman** [comment]
Yes (actually, it's in a .tar file, so I'm extracting it, parsing it, then
saving it to a database.

I'm beginning to think that the issue is that for my data, I don't have an
rdf spec.  That is, the pgterms namespace isn't valid for

http://www.gutenberg.org/cache/epub/10006/pg10006.rdf

Thanks for your help.  I'm soooo confused about namespaces, I feel like I'm
hacking a lot to get the title, subject codes and other basic data for
these files.
